packages/amplify_auth_plugin_interface/lib/src/SignUp/ResendSignUpCodeRequest.dart

```diff
@@ -13,14 +13,12 @@
  * permissions and limitations under the License.
  */
 
-import 'package:flutter/foundation.dart';
 
-import './SignUpOptions.dart';
 
 class ResendSignUpCodeRequest {
   String username;
 
-  ResendSignUpCodeRequest({@required this.username,});
+  ResendSignUpCodeRequest({required this.username,});
```

Comment:
probably don't need a `,` in the end. I know not your code but just a good cleanup

Reply:
Sounds good